students/seungho-shin/user/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from user.views import SignupView
+
+urlpatterns = [
+    path('', SignupView.as_view())
```

Comment:
```suggestion
    path('/signup', SignupView.as_view())
```
RESTfull API에서 더 자세히 배우겠지만, 어떤 기능 또는 자원을 사용하는지를 명시하는 것이 좋습니다.